### PR TITLE
Refactored with Hooks

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I implemented the following:

- Refactored the Calculator component (components/Calculator.js) from a class-based component to a functional component with hooks.
- Tested the app to make sure it still works fine.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,63 +1,52 @@
...
-         <div className="result">
-           {total}
-           {' '}
-           {operation}
-           {' '}
-           {next}
-         </div>
-         <button type="button" onClick={this.handleClick} className="greyBtn">AC</button>
-         <button type="button" onClick={this.handleClick} className="greyBtn">+/-</button>
-         <button type="button" onClick={this.handleClick} className="greyBtn">%</button>
```
Comment (henatan99) [inline src/components/Calculator.js:35]: - [ ] You did a great job refactoring the class-based Calculator component to a functional component with hooks. But you should make sure that there is no browser or console error. When I hit an operator button multiple times consecutively it throws a browser error. Kindly fix that. 

Screenshot from 2021-09-16 14-51-03

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @themonster2015 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (themonster2015) [reply src/components/Calculator.js:35]: thanks. I've fixed it

**Review by AshakaE — APPROVED** [review]
### Hi @themonster2015  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**